test/cli/autocorrect-private-not-suggested/autocorrect-private-not-suggested.sh

```diff
@@ -0,0 +1,7 @@
+# verifies that the file is unchanged when suggestions are applied
+# i.e. it does not replace the unknown method with a private one
+tmp=$(mktemp -d)
+cp test/cli/autocorrect-private-not-suggested/*.rb "$tmp"
+main/sorbet --silence-dev-message -a "$tmp"/*.rb 2>/dev/null
+cat "$tmp"/*.rb
```

Comment:
For simplicity, can you combine both of these tests into a single CLI test, and make that one CLI test output both the errors and the file post-autocorrect?

If you're looking for a test to cargo cult, this is a good one

https://github.com/sorbet/sorbet/blob/master/test/cli/suggest_t_must/suggest_t_must.sh